src/app/shared/palette/document/write-document-field.html

```diff
@@ -8,6 +8,7 @@
 
   <div>
     <ccd-read-document-field *ngIf="caseField" [caseField]="caseField"></ccd-read-document-field>
+    <div class="chosen-file" *ngIf="selectedFile && selectedFile.name && caseField.value">Chosen file: {{ caseField.value }}</div>
```

Comment:
can we simplify this to:
`<div class="chosen-file" *ngIf="selectedFile">Chosen file: {{ selectedFile.name }}</div>`
If not, can we move this long if condition inside a method in the component called something like 'isFileUploaded()'